picard/webservice.py

```diff
@@ -314,22 +310,22 @@ def _process_reply(self, reply):
             reply.deleteLater()
 
     def get(self, host, port, path, handler, xml=True, priority=False,
-            important=False, mblogin=False, cacheloadcontrol=None, refresh=False):
+            important=False, mblogin=False, cacheloadcontrol=None, refresh=False, queryargs=None):
```

Comment:
I think we should define query object and use it instead of passing so many parameters.
```python
query_obj = QueryObj(host, port, path, handler)
query_obj.args = { 'key': 'value' }
ws.get(self, query_obj)
````